download_data.sh

```diff
@@ -128,7 +128,7 @@ function download_wbless () {
 function download_eval () {
     echo -e 'word1\tword2\tlabel\trelation\tfold'
     curl $CURL_OPTIONS "$VERED_REPO_URL/EVALution.val" "$VERED_REPO_URL/EVALution.test" | \
-        sort | uniq | sed 's/-[jvn]\t/\t/g' | \
+        sed 's/-[jvn]\t/\t/g' | sort | uniq | \
```

Comment:
So I don't want to accept the strip-pos tags part of this patch, because I want to keep this repo aligned with the results of the paper for replication reasons. Your proposal is reasonable and you shouldn't feel hindered from doing this in your own research.

Reply:
I've reversed this now.